apps/web/src/utils/number.ts

```diff
@@ -61,6 +48,34 @@ export function formatNumber(
   );
 }
 
+export function removeCommas(formattedNumber: string): string {
+  return formattedNumber.replace(/,/g, "");
+}
+
+export function parseSuffixedNumber(value: string): [number, string?] {
+  // Remove any leading/trailing whitespace
+  value = removeCommas(value.trim());
```

Comment:
I would move the `trim()` operation to `removeCommas()` as we are performing the operation again on `L74`. This way, we can pass any value and forget about the whitespace removal